src/endpoints/base.ts

```diff
@@ -1,4 +1,4 @@
-import Api from '../api';
+import {Api} from '../api';
```

Comment:
I think there should be some spacing like this `import { Api } from '../api';`. Maybe try running the formatter?

Same for `TMDB` in `src/index.ts`